prime-router/src/main/kotlin/tokens/Server2ServerAuthentication.kt

```diff
@@ -38,7 +37,7 @@ class Server2ServerAuthentication(val workflowEngine: WorkflowEngine) : Logging
      * Data class that holds the data from a parsed JWT.  The organization and sender fields are derived from
      * the issuer in the JWT claims
      */
-    data class ParsedJwt(val organization: Organization, val sender: Sender?, val kid: String?, val kty: KeyType)
+    data class ParsedJwt(val organization: Organization, val kid: String?, val kty: KeyType)
```

Comment:
We no longer need access to the sender if that happens to be what `iss` maps too now that all the keys are on the organization